languages/csharp/exercises/concept/randomness/.meta/design.md

```diff
@@ -0,0 +1,19 @@
+## Learning objectives
+
+- Know how to implement randomness with `System.Random` in C#.
+- Know that no seed is required
+- Know that both integers and real numbers can be generated
+- Know that once generated random numbers can be easily used for many purposes (including non-numeric ones).
+
+## Out of scope
+
+- Secure random numbers
+
+## Concepts
+
+- `randomness`: as they are handled in C# and .NET.
```

Comment:
```suggestion
- `randomness`: know how to implement randomness with `System.Random` in C#; know that no seed is required; know that both integers and real numbers can be generated; know that once generated random numbers can be easily used for many purposes (including non-numeric ones)
```

Reply:
Mmm.  I have seen that a number of the concepts descriptions follow this pattern.  I feel that this is a description of the exercise not the concept.

What can we say about randomness?  I don't think we want so much to attempt a definition of randomness but perhaps to put it in the context of C# and (as it revolves) round a library API) get the student in the habit of thinking about _.NET_ as a thing.

Reply:
> I have seen that a number of the concepts descriptions follow this pattern. I feel that this is a description of the exercise not the concept.

It actually should be sort of a combination: what are the things about the Concept that a student will know after completing the exercise. Does that make more sense?

Reply:
I suppose.  Concept == chunk of knowledge.  Naming really is very difficult.